backend/tests/Feature/Web/Admin/ProcessTemplateEditGuardTest.php

```
@@ -0,0 +1,140 @@
+            'action' => 'template_step.added',
+        ]);
+    }
+
+    public function test_updating_a_step_records_before_and_after(): void
+    {
+        $step = $this->step(1, 'Original');
+
+        $this->actingAs($this->admin)->put("{$this->base()}/steps/{$step->id}", ['name' => 'Renamed'])
+            ->assertRedirect();
+
+        $log = AuditLog::where('action', 'template_step.updated')->latest('id')->first();
```

PHPStan (2.2.8)

[error] Call to an undefined static method App\Models\AuditLog::where().


(staticMethod.notFound)